packages/@webex/plugin-meetings/test/unit/spec/multistream/mediaRequestManager.ts

```diff
@@ -552,28 +552,28 @@ describe('MediaRequestManager', () => {
     ]);
   });
 
-  it('clears all the requests on reset()', () => {
+  it('avoids sending duplicate requests and clears all the requests on reset()', () => {
```

Comment:
I'm not entirely sure this is how it's supposed to work.
The `addRequest` method has a flag (`commit`) which forces the method to also send the request immediately after adding it. So this test was initially testing that, after manually triggering commit after each addRequest (see how the last param used to be `true` and now is `false`), and calling commit again, it will re-send all the requests again (in a batch). 

Is this behaviour necessary still? I.e forcing the requests to be send on `addRequest` and then sending them again as a batch? At the moment this is no longer the case, because it detects them as duplicates.

It's also why i've slightly changed the meaning of this test.

Reply:
yes the behavior is needed @marcin-bazyl

Reply:
@Coread